slices/ruby-webrick.yaml

```
@@ -0,0 +1,14 @@
+package: ruby-webrick
+
+essential:
+  - ruby-webrick_copyright
+
+slices:
+  rubygems:
+    contents:
+      /usr/share/rubygems-integration/all/gems/webrick-1.8.1/**:
+      /usr/share/rubygems-integration/all/specifications/webrick-1.8.1.gemspec:
```

Same comment from `slices/ruby-net-telnet.yaml`
```suggestion
      /usr/share/rubygems-integration/all/gems/webrick-*/**:
      /usr/share/rubygems-integration/all/specifications/webrick-*.gemspec:
```

---

done in 836e456